### DISCUSSION TITLE
Installation issues:  postinstall.js not found

### DISCUSSION THREAD

**Discussion by markyarnold** [serverless/serverless#11329] [Q&A]
Hi all;

Trying to run npm i -g serverless and it fails, irrespective of Node version as follows:

<img width="881" alt="image" src="https://user-images.githubusercontent.com/56762387/184123375-ad920ecf-0e6f-4aae-84dc-2d104c6d7d75.png">

Tried everything; I'm obviously missing something!  Any idea's?

**Comment by mnapoli** [comment]
I just ran `npm -g i serverless` without problems. Could it be your NPM version maybe?
